### ISSUE TITLE
[a11y audit] Testing: Chatbot MHV Prescription Skill 4.2.12: Supplement text with icons

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#67286]
## 4.2.12 Supplement test with icons
- **Testing Status:** `Completed`
- **Findings**: `Present`
- **Audit Spreadsheet:** [Link to 4.2.12](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987)

## Testing Detail 

I reviewed the prescription skill for the following:

> How might colors and symbols complement the text for increased comprehension? When used appropriately, graphics can make text content easier to read and understand for some users. Try to identify any text content that could be made easier to read through visual cues.

### Suggestion for improved comprehension 🫂  

When someone submits a refill using the chatbot, it would be helpful to have a visual cue for whether the submission went through (was submitted) or didn't (wasn't submitted, for whatever reason). 

### Recreating the Issue 🖥️ 

- Access the prescription skill
- Select "Request refills"
- Request a refill

This will result in a message:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/907a358e-455d-4728-8129-ea4dce2047e6)

### How to implement 🛠️ 

In the "Refill submitted." message, you could add an accessible icon of a checkmark.

While I haven't encountered it, I'm assuming there's a similar message for when the submission has failed. In that case, add an accessible icon of an X.

Example:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/b3858c7e-5fa4-465f-a605-d1bf87b77836)

You can use colors for these icons, as long as they pass color contrast. :)

You can [follow this guidance for creating accessible icons](https://a11y-101.com/development/icons-and-links).
